src/pages/map/map.ts

```diff
@@ -22,12 +24,12 @@ export class MapPage {
   }
 
   initNativeMaps(mapEle) {
-    this.map = new GoogleMap(mapEle);
+    this.googleMap = new GoogleMap(mapEle);
     mapEle.classList.add('show-map');
 
-    GoogleMap.isAvailable().then(() => {
-      const position = new GoogleMapsLatLng(43.074395, -89.381056);
-      this.map.setPosition(position);
+    this.googleMaps.isAvailable().then(() => {
```

Comment:
Looks like not a typo, or there is something wrong with the typescript intellisense. I will leave it as is and someone with more knowledge should check.

![capture064](https://cloud.githubusercontent.com/assets/140737/25111443/e108dbd4-239f-11e7-8927-f0c054c626c3.png)